docs/code-of-conduct-response.rst

```diff
@@ -1,76 +1,121 @@
 
-Code of Conduct Response Playbook
-=================================
+Code of Conduct Response Guide
+==============================
 
-This document aims to help organizers and moderators of Write the Docs spaces to handle reports of :doc:`code-of-conduct` violations.
+.. contents::
+   :local:
+   :depth: 2
+   :backlinks: none
 
-CoC violations at conferences
------------------------------
+This document aims to help organizers and moderators of Write the Docs spaces to handle reports of :doc:`code-of-conduct` reports.
+This includes conferences, online spaces and meetups.
 
-Presentations
-~~~~~~~~~~~~~
-Presentations or similar events should not be stopped for one-time gaffes or minor problems, although a member of conference staff should speak to the presenter afterward. However, staff should take immediate action to politely and calmly stop any presentation or event that repeatedly or seriously violates the anti-harassment policy. For example, simply say "I'm sorry, this presentation cannot be continued at the present time" with no further explanation.
+General guidelines for Code of Conduct response
+-----------------------------------------------
 
-In-person reports
-~~~~~~~~~~~~~~~~~
+The role Code of Conduct response teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When taking a report from someone experiencing harassment you should record what they say and reassure them they are being taken seriously, but avoid making specific promises about what actions the organizers will take.
+Different spaces have different response teams to handle Code of Conduct reports:
 
-Ask for any other information if the reporter has not volunteered it (such as time, place) but do not pressure them provide it if they are reluctant. Even if the report lacks important details such as the identity of the person taking the harassing actions, it should still be recorded and passed along to the appropriate staff member(s).
+* Each Write the Docs conference has a Code of Conduct team.
+* For online platforms, reports are made to conduct@writethedocs.org.
```

Comment:
I'd love to see a more formal designation of moderators/staff/core on Slack (in PyCon Balkan for example they suffixed "ORG" to the Slack nicknames so you see who's on the org team/mods, makes it more transparent and easier to find/contact people. This is more of a note for myself for the org guide but maybe we can circulate this as a part of the CoC revamp

Reply:
Yes, I think this is good. But let's not include it in the scope of this task now.